client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -35,6 +35,7 @@ import support from 'lib/url/support';
 import { registrar as registrarNames } from 'lib/domains/constants';
 import DesignatedAgentNotice from 'my-sites/upgrades/domain-management/components/designated-agent-notice';
 import Dialog from 'components/dialog';
+import { getCurrentUser } from 'state/current-user/selectors';
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?